optimum/habana/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -561,6 +576,8 @@ def __call__(
                 if not self.use_hpu_graphs:
                     self.htcore.mark_step()
 
+            hb_profiler.step()
```

Comment:
Should be `stop` instead of `step` here right?
```suggestion
            hb_profiler.stop()
```

Reply:
thank you for catching the bug.